### ISSUE TITLE
Follows the price when the product it's favorite and not finalized.

### ISSUE THREAD

**Issue by drielnox** [MercadoTrack/extension-local#9]
This feature maybe can be a optional setting, part of extension.

**Comment by niconj** [comment]
I don't know if I got this issue right, MT already follows the price of the article as long as the article is set to "follow" by the user. Do you mean that it should be automatically un-followed if the article publication becomes finalized?

**Comment by drielnox** [comment]
No. The desired behavior is when you press the favorites button in ML, automatically, MT follow the price of the publication.

The other desired behavior is that if the publication is finished and the price is being followed, stop following because it does not make sense to continue doing so.

PS: Does it make sense that we continue to write in English? I know that you are Argentinian, I am Argentine...

**Comment by GMaiolo** [comment]
Me gusta la idea de trackear cuando se agrega a favoritos.
Qué pasaría si lo saco de favoritos, lo sigo trackeando o lo deja de trackear?

**Comment by drielnox** [comment]
Si me pongo en la piel del usuario, una de las principales razones de no seguir más algo es porque ya no me interesa comprarlo, por ende, no tendría sentido seguirle el precio.

¿Por que otro motivo sacaria mi favorito de una publicacion?

La única razón que veo que se siga el precio despues de sacarle al favorito es por que empece a seguir el precio por fuera de la funcionalidad favorito. O sea, le di al "Seguir precio" sin tocar el favorito. Si hago eso, es por que quiero seguir el precio del producto sin querer comprar, onda... como para hacer un estudio de mercado y evolución de precios. Ahi la herramienta se tendría un propósito totalmente diferente al pensado originalmente.
